### DISCUSSION TITLE
Snap

### DISCUSSION THREAD

**Discussion by kx32** [helgeerbe/certbot-dns-ionos#14] [Q&A]
Is there a way of getting the plugin to work with Snap since that is the preferred installation for certbot.

**Comment by helgeerbe** [comment]
Hi @kx32 I never used Snap. But I found this for hetzner dns plugin. It seems not too difficult to provide a snap package. I have to think about it. But this will take some time.

**Comment by ph-kvieth** [comment]
I would like to have it as snap too

**Comment by helgeerbe** [comment]
Does anybody can try this guide? https://github.com/certbot/certbot/tree/master/tools/snap
In the moment I have limited time. But I can publish a snap if I get some support.
